### DISCUSSION TITLE
install failed with attempts

### DISCUSSION THREAD

**Discussion by Stars-22** [morrownr/8814au#87] [Q&A]
Running install-driver.sh version 20220705
5.15.0-47-generic
x86_64
Starting installation...
Copying source files to: /usr/src/rtl8814au-5.8.5.1
Copying 8814au.conf to: /etc/modprobe.d
Creating symlink /var/lib/dkms/rtl8814au/5.8.5.1/source -> /usr/src/rtl8814au-5.8.5.1

Kernel preparation unnecessary for this kernel. Skipping...

Building module:
cleaning build area...
'make' -j4 KVER=5.15.0-47-generic KSRC=/lib/modules/5.15.0-47-generic/build...(bad exit status: 2)
ERROR (dkms apport): binary package for rtl8814au: 5.8.5.1 not found
Error! Bad return status for module build on kernel: 5.15.0-47-generic (x86_64)
Consult /var/lib/dkms/rtl8814au/5.8.5.1/build/make.log for more information.
An error occurred. dkms build error = 10
Please report this error.
You will need to run the following before reattempting installation.
$ sudo ./remove-driver.sh

-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
bcmwl/6.30.223.271+bdcom, 5.15.0-47-generic, x86_64: installed
rtl8814au/5.8.5.1: added
------------------------------------------------------------------------------------------------------
I follow the steps strictly,but I can't complete it.I need help!

**Comment by morrownr** [comment]
Hi @Stars-22 

What distro and version are you running?

How about we do this to see what happens:

$ sudo ./remove-driver.sh
$ make clean
$ make
$ sudo make install
$ sudo reboot
